### ISSUE TITLE
Description of functions?

### ISSUE THREAD

**Issue by carlos1w** [adafruit/Adafruit_HMC5883_Unified#14]
Is there a description of the functions of this library besides the example?  I appreciate your effort creating this library, but there should be a description of what the functions are in a readme file.

**Comment by manitoulin-maker** [comment]
I AGREE, BEEN STUCK, JUST HIT EVERY LETTER TILL YOU READ EVERY OPTION I GUESS LOL
